### PR TITLE
Added the ability to have absolute Paths

### PR DESCRIPTION
**Pull request by benwestrate**
Hey @rupurt, thanks for your work on this plugin it was almost exactly what I needed just had to add one small thing. 
Created an options object containing an attribute `absoluteMappingPaths` which when set to true the plugin will now assume that you are giving it the absolute path to the destination. This gives the plugin a bit more flexibility.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,6 +30,10 @@ WebpackCopyAfterBuildPlugin.prototype.apply = function(compiler) {
         var from = outputPath + "/" + chunkFilename;
         var to = outputPath + "/" + mapping;
 
+        if( options.absoluteMappingPaths === true ){
```
Comment (rupurt) [inline index.js]: How do you feel about using truthyness here and only assigning the other `to` in the `else` condition? i.e.

```javascript
var to;

if (options.absoluteMappingPaths) {
   to = mapping;
} else {
   to = outputPath + "/" + mapping;
}
```

**Review by rupurt — CHANGES_REQUESTED** [review]

**Comment by rupurt** [comment]
Hey @benwestrate,

Thanks for the PR! I added a minor comment.

File: index.js (hunk 1 above)
Comment (benwestrate) [reply index.js]: sure sounds good ill make the change

Comment (benwestrate) [reply index.js]: Ok, change made. Let me know if there is anything else that stands out.

Comment (rupurt) [reply index.js]: Sweet. Thanks @benwestrate

**Comment by benwestrate** [comment]
@rupurt could you also publish this back to npm? thanks!

**Pull request merged**